marshmallow/utils.py

```diff
@@ -334,16 +334,31 @@ def callable_or_raise(obj):
     return obj
 
 
-def get_func_args(func):
```

Comment:
Thanks. 

I was suggesting to rebase this branch onto the `issue540` branch, so that the test will be included with this branch/PR. Would you mind doing that?